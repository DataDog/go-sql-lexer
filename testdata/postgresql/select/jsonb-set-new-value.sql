SELECT jsonb_set(data, '{user,name}', '"John Doe"') AS updated_data FROM user_profiles;
