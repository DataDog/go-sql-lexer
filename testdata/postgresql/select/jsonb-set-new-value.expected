[
    {
        "expected": "SELECT jsonb_set ( data, ?, ? ) FROM user_profiles",
        "statement_metadata": {
            "size": 19,
            "tables": [
                "user_profiles"
            ],
            "commands": [
                "SELECT"
            ],
            "comments": [],
            "procedures": []
        }
    },
    {
        "expected": "SELECT jsonb_set(data, ?, ?) FROM user_profiles",
        "normalizer_config": {
            "remove_space_between_parentheses": true
        }
    }
]